summaries/GameGAN.md

```diff
@@ -0,0 +1,97 @@
+
+<img src='../images/GameGAN_memory_eqn.png'>
+
+where; *K*, *G* and *E* are small MLPs. *w* is a learned shift kernel that depends on the current action, and the kernel is used to shift *α<sub>t−1</sub>*.
+
+###Rendering Engine
+
+- The (neural) rendering engine is responsible for rendering the simulated image *x<sub>t+1</sub>* given the state *h<sub>t</sub>* . It can be simply implemented with standard transposed convolution layers.
+
+- The paper also mentions a specialized rendering engine architecture for ensuring long-term consistency by learning to produce disentangled scenes.
+ 
+## Proposed Loss functions
+
+- **Adversarial Losses**: There are three main components:
+
+    - **Single image discriminator**: To ensure each generated frame is realistic, the single image discriminator and GameGAN simulator play an adversarial game.
+    
+    - **Action-conditioned discriminator**: GameGAN has to reflect the actions taken by the agent faithfully. We give three pairs to the action-conditioned discriminator. The job of the discriminator is to judge if two frames are consistent with respect to the action. Therefore, to fool the discriminator, GameGAN has to produce realistic future frame that reflects the action.
+    
+    - **Temporal discriminator**: Different entities in an environment can exhibit different behaviors, and also appear or disappear in partially observed states. To simulate a temporally consistent environment, one has to take past information into account when generating the next states. Therefore, we employ a temporal discriminator that is implemented as 3D convolution networks. It takes several frames as input and decides if they are a real or generated sequence.
+
+- **Cycle Loss**
+
+    - To ensure long-term consistency of static elements, the memory module and the rendering engine are leveraged to disentangle static elements from dynamic elements.
+
+    - After running through some time steps *T* , the memory block *M* is populated with information from the dynamics engine. 
+    
+    - Using the memory location history *α<sub>t</sub>* , we can retrieve the memory vector *m̂<sub>t</sub>* which could be different from *m<sub>t</sub>* if the content at the location *α<sub>t</sub>* has been modified. 
+    
+    - Now, *c = { m̂ t , 0}* is passed to the rendering engine to produce *X<sup>m̂<sub>t</sub></sup>* where *0* is the zero vector and *X<sup>m̂<sub>t</sub></sup>* is the output component corresponding to *m̂<sub>t</sub>*. We use the following loss:
```

Comment:
Here *c = { m̂ t , 0}*, shouldn't it be *c = { m̂<sub>t</sub> , 0}*, I am not sure though. Please check once.

Reply:
Yes, you are right. Didn't add the <sub> tag there. Sorry!